### PR TITLE
Sort output of listfacts and listdetailedfacts

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: plugins/factoids.py [hunk 1]
```diff
@@ -187,6 +187,6 @@ def listfactoids(notice, chan):
 def listdetailedfactoids(chan):
     """- lists all available factoids with their respective data"""
     headers = ("Command", "Output")
-    data = [(fact[0], fact[1]) for fact in factoid_cache[chan].items()]
+    data = [(fact[0], fact[1]) for fact in sorted(factoid_cache[chan].items())]
     table = gen_markdown_table(headers, data).encode('UTF-8')
```
Comment (linuxdaemon) [inline plugins/factoids.py]: `gen_markdown_table` is undefined at this point

Comment (ghost) [reply plugins/factoids.py]: I can't reproduce that error.

**Comment by ghost** [comment]
Bump.

**Comment by ghost** [comment]
Thank you.

**Pull request merged**